app/controllers/accounts_controller.rb

```diff
@@ -1,9 +1,15 @@
 class AccountsController < ApplicationController
   before_action :set_account, only: %i[ show edit update destroy ]
+  protect_from_forgery with: :null_session
+  before_action :authenticate_user!
+  
+
 
   # GET /accounts or /accounts.json
   def index
-    @accounts = Account.all
+    @accounts = Account.where(user_id: current_user.id).sort_by(&:id)
```

Comment:
I think that by default these should come back sorted by `id`. I'm curious about your reasoning for adding the sort here.

Reply:
Good question! Looking at this code, I don't quite remember...I'll remove that sort method from this